core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLTPFSTestRun.java

```diff
@@ -310,8 +310,8 @@ public void testAvroSourceConversionToAvroSink() throws Exception {
     dataFileWriter.flush();
 
     txTpfs.commitTx();
-    txService.canCommit(tx1, txTpfs.getTxChanges());
-    txService.commit(tx1);
+    txService.canCommit(tx1.getTransactionId(), txTpfs.getTxChanges());
```

Comment:
we should remove usage of TransactionManager completely and just call fileSetManager.flush().

Reply:
agreed, will push another commit